tds/src/main/java/thredds/server/config/TdsContext.java

```diff
@@ -506,102 +459,4 @@ public String getTdsDebugFlags() {
   public void setContentRootPathProperty(String contentRootPathProperty) {
     this.contentRootPathProperty = contentRootPathProperty;
   }
-
-  public void setUploadDirProperty(String uploadDirProperty) {
```

Comment:
technically this is public... so maybe it is a breaking change?

Reply:
I guess technically that would be more responsible, but I'm inclined to just delete them. We've already broken the functionality by ripping it out if anyone is using them, so even if they were left in and deprecated they'd just be no-ops. And that's only if someone is plugging into the TDS....

Reply:
Okay, if you are okay with it then we can just allow the breaking change.

Reply:
ugh, no, we should be responsible and deprecate them and let them no-op. If we remove them and anyone is using them (so unlikely though!) it could crash the whole application instead of just not supporting uploads :/

Reply:
okay, sounds good!

Reply:
should be fixed now!